index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <link rel="stylesheet" type="text/css" href="stylesheets/style.css" />
+    <link rel="stylesheet" type="text/css" href="stylesheets/reset.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Mint's SignUp Page</title>
+  </head>
+
+  <body>
+    <div class="main-container-div">
+      <header class="nav-bar-div">
+        <div class="nav-bar-text-div">
+          <p>
+            Already have an Intuit account? <br />
+            <span>I forgot my User ID or Password</span>
+          </p>
+        </div>
+        <div>
+          <button class="sign-up-btn-div">
+            <i class="fa fa-lock"></i> Sign In
+          </button>
+        </div>
+      </header>
+      <main class="logo_and_form_main">
+        <div class="logo-div">
+          <img src="icons/logo.PNG" alt="mint logo" />
+        </div>
+        <div class="form-div">
+          <form class="form-section">
+            <div class="form-text-div">
+              <h3>Create an Intuit account</h3>
+              <p>One account for everything Intuit,including Mint.</p>
+              <a href="#">Learn more</a>
+            </div>
+            <div class="input-boxes-div">
+              <fieldset>
+                <legend>Email address</legend>
+                <input type="email" name="email" id="email" />
+              </fieldset>
+              <fieldset>
+                <legend>Phone (recommended)</legend>
+                <input type="tel" name="phone" id="phone" />
+                <p>Standard call messaging or data rates may apply</p>
+              </fieldset>
+              <fieldset>
+                <legend>Password</legend>
+                <input type="password" name="password" id="password" />
+              </fieldset>
+              <fieldset>
+                <button class="input-btn-div">
+                  <i class="fa fa-lock"></i> Create Account
+                </button>
```

Comment:
[REQUIRED] Type should be 'submit' for the button. [reference](https://www.w3schools.com/tags/att_input_type_submit.asp)